### PR TITLE
chore: align tailwind tables with ui design

### PR DESCRIPTION
**Pull request by connoratrug**
What are the main changes you did:
- Align tabels page with [design](https://www.figma.com/proto/uSAeYVHKsOIa2BXT7Kx07n/Molgenis-Design?page-id=868%3A1678&node-id=4412-335236&node-type=frame&viewport=26035%2C-8351%2C0.5&t=VWhbD6hYkc8lCC6G-1&scaling=scale-down-width&content-scaling=fixed&starting-point-node-id=4412%3A335236&show-proto-sidebar=1)
- remove content block container 
- add bg color to table 
- add border radius to table
- set margen as per design
- remove debug loading indicator

how to test:
- explain here what to do to test this (or point to unit tests)

todo:
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

**Review by mswertz — COMMENTED** [review]
what to do with rounded borders limiting text?
image

**Review by mswertz — APPROVED** [review]
ok, we solve the issue with rounded corners later

File: apps/tailwind-components/components/table/TableEMX2.vue [hunk 1]
```diff
@@ -64,67 +64,61 @@ function handlePagingRequest(page: number) {
...
-                : 'none'
-            "
-            scope="col"
-          >
-            <button
-              class="overflow-ellipsis whitespace-nowrap overflow-hidden hover:cursor-pointer"
-              @click="handleSortRequest(column.id)"
+
+  <div class="overflow-auto rounded-b-50px">
+    <div class="overflow-x-auto overscroll-x-contain bg-table rounded-t-3px">
```
Comment (davidruvolo51) [inline apps/tailwind-components/components/table/TableEMX2.vue:69]: To have enough spacing at the bottom, it's possible to add `pb-*` here. In this design, I noticed that the text hugs the left side of the table. We could consider adding a little bit of space in the first cell in each row.

File: apps/tailwind-components/components/table/TableEMX2.vue [hunk 2]
```diff
@@ -64,67 +64,61 @@ function handlePagingRequest(page: number) {
...
+                    column.id === settings.orderby.column &&
+                    settings.orderby.direction === 'DESC'
+                  "
+                  class="w-4 h-4 inline-block"
+                />
+              </button>
+            </th>
+          </tr>
+        </thead>
+        <tbody class="mb-3">
```
Comment (davidruvolo51) [inline apps/tailwind-components/components/table/TableEMX2.vue:106]: To remove the border from the last row, you can structure the class like this -

```
<tbody class="mb-3 [&_tr:last-child_td]:border-none">
```

I found that the text feels compressed. We could also add a little bit of space on the bottom: 

```
<tbody class="mb-3 [&_tr:last-child_td]:border-none [&_tr:last-child_td]:mb-5">
```

**Review by davidruvolo51 — CHANGES_REQUESTED** [review]
Looks great!

I added a suggestion the rounded table border and spacing issue in the last row.

**Review by davidruvolo51 — APPROVED** [review]
Looks good!

**Pull request merged**